modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -49,6 +49,8 @@ Before upgrading and testing your custom connector for Java 17, you must be fami
 NOTE: The maximum supported version for running your applications is Java 17 for Mule runtime 4.6.x and later. During the packaging of your app, all code, including third-party dependencies, *must* be compiled for Java 8.
 +
 Because you must compile your custom connector with Java 8, use the Java compiler configuration from the Java SDK parent POM. You can’t use language features from Java 11 or Java 17. 
++
+If you change the compilation to Java 17, the surrounding tooling (such as Studio, Maven plugins, DataSense, connectivity testing, etc) breaks because they are still running on Java 8.
```

Comment:
```suggestion
If you change the compilation to Java 17, the surrounding tooling (such as Studio, Maven plugins, DataSense, connectivity testing, etc.) breaks because they are still running on Java 8.
```

Reply:
There's no reference to the use of "etc." in the MuleSoft or Salesforce guide. Weird...

Reply:
Seems like there is: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=etc.%20%3E%20x%2C%20y%2C%20or%20z%2C%20and%20so%20on

I just updated the wording